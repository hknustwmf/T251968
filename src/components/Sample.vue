<template>
	<div class="sample">
		<h1>Sampel Header</h1>
	</div>
</template>

<script lang="ts">
import Vue from 'vue';
import Component from 'vue-class-component';

@Component
export default class Sample extends Vue {
	created() {
		console.log( 'abc' );
	}
}
</script>
