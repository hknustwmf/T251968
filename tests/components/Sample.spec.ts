import Sample from '@/components/Sample.vue';
import { shallowMount } from '@vue/test-utils';

describe( 'Sample', () => {
	it( 'renders correctly', () => {
		const wrapper = shallowMount( Sample );
		expect( wrapper.element ).toMatchSnapshot();
	} );

	it( 'mounted correctly', () => {
		const wrapper = shallowMount( Sample );
		expect( wrapper.isVueInstance() ).toBe( true );
	} );
} );
