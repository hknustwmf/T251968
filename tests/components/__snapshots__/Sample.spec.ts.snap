// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sample renders correctly 1`] = `
<div
  class="sample"
>
  <h1>
    Sample Header
  </h1>
</div>
`;
